ioctl/cmd/ws/wsproject.go

```diff
@@ -77,6 +99,11 @@ func init() {
 		config.TranslateInLang(_flagProjectRegisterContractAddressUsages, config.UILanguage),
 	)
 	_ = wsProject.MarkFlagRequired("contract-address")
+
+	if wsProjectIPFSEndpoint == "" {
+		wsProjectIPFSEndpoint = "localhost:5001"
+		wsProjectIPFSGatewayEndpoint = "127.0.0.1:8080"
```

Comment:
Any reasons for one is `localhost`, another is `127.0.0.1`?

Reply:
ipfs endpoint for uploading and gateway for fetching